models/balances/bitcoin/balances_bitcoin_sataoshi_latest_day.sql

```diff
@@ -0,0 +1,41 @@
+{{ config(
+        schema = 'balances_bitcoin',
+        alias = 'satoshi_latest_day',
+        materialized='view',
+
+        post_hook='{{ expose_spells(\'["bitcoin"]\',
+                                        "sector",
+                                        "balances",
+                                        \'["gandalf"]\') }}'
+        )
+}}
+
+WITH
+      updated_balances as (
+            SELECT
+                blockchain
+                , day
+                , wallet_address
+                , amount_raw
+                , amount
+                , price_btc
+                , profit
+                , amount_usd
+                , total_asset
+                , row_number() OVER (partition by wallet_address order by day desc) as latest_balance
+            FROM {{ ref('balances_bitcoin_satoshi_day') }}
+      )
+
+SELECT
+    blockchain
+    , day
+    , wallet_address
+    , amount_raw
+    , amount
+    , price_btc
+    , profit
+    , amount_usd
+    , total_asset
+    , now() as updated_at
```

Comment:
this likely won't help you on either spell, as it's a view and will calculate `now()` every time you execute a query on it in the app

Reply:
we can leave as-is for now, then if you revisit to redesign these spells later, it'll be there as a reminder to include

Reply:
Thanks for pointing that out! Yes, let’s do that and I will refactor this later